android/build.gradle

```diff
@@ -6,10 +6,7 @@ buildscript {
         compileSdkVersion = 28
         targetSdkVersion = 28
         glideVersion = "4.9.0"
-        // googlePlayServicesVersion = "17.0.0"
-        // supportLibVersion = "1.0.2"
-        // mediaCompatVersion = '1.0.1'
-        // supportV4Version = '1.0.0'
+        kotlin_version = '1.3.21'
```

Comment:
Where is Kotlin being used here? Try using a newer version (aka 1.3.50 at the time of this comment).

Reply:
Shouldn't better to declare it like `kotlinVersion` instead? This way we keep the same naming "convection" being used here to declare the libraries version.